frontend/packages/shared/defi/unit.ts

```diff
@@ -8,20 +8,21 @@ export function toChainIdUnit(value: number | BigNumber, decimalPlaces = 12) {
   return bigNumberValue.multipliedBy(10 ** decimalPlaces);
 }
 
-export function fromChainIdUnit(value: number | BigNumber | BigInt, decimalPlaces = 12) {
-  return (BigNumber.isBigNumber(value) ? value : new BigNumber(value.toString())).dividedBy(
-    10 ** decimalPlaces
-  );
+export function fromChainIdUnit(
```

Comment:
Historical values coming from Subsquid are currently already doing this conversion, so you will not always need it.

Reply:
Thanks for letting me know. But this is a utility function we use to convert all values from on chain.